src/externs/datatables.js

```diff
@@ -3,6 +3,22 @@
  */
 
 /**
+ * @constructor
+ * @return {!dataTable}
+ */
+function dataTable() {}
```

Comment:
Based on DataTables convention (http://datatables.net/reference/type/DataTables.Api), "DataTables" is a constructor type. "DataTables.Api" is a namespaced type. Therefore it is sth like:
```js
function DataTables() {}

/**
 * @constructor
 */
DataTables.Api = function() {};

/**
 * @return {DataTables.Api}
 */
DataTables.prototype.rows = function(...) {};
```